techiteasy.sql

```diff
@@ -0,0 +1,136 @@
+-- Database: TechItEasy
+
+-- DROP DATABASE IF EXISTS "TechItEasy";
+
+CREATE DATABASE "TechItEasy"
+    WITH
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'nl_NL.UTF-8'
+    LC_CTYPE = 'nl_NL.UTF-8'
+    LOCALE_PROVIDER = 'libc'
+    TABLESPACE = pg_default
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
+
```

Comment:
Ik kon jouw sql bestand niet meteen uitvoeren, misschien heb je hem in andere software geschreven?
pgAdmin had moeite met het CREATE DATABASE deel, toen ik dat heb uitgeschakeld deed hij het wel